packages/expo/src/Linking/LinkingModule.web.ts

```diff
@@ -0,0 +1,40 @@
+import NativeEventEmitter from 'react-native/Libraries/EventEmitter/NativeEventEmitter';
```

Comment:
See comment below about why we might not want to use NativeEventEmitter at all, but also wanted to point out that expo-core exports an arguably better NativeEventEmitter that smooths over differences between Android and iOS's event listening systems.